app/Console/Command/RejectPlaceDeprecatedType.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Model\ValueObject\Identity\ItemIdentifier;
+use CultuurNet\UDB3\Place\Commands\Moderation\Reject;
+use CultuurNet\UDB3\Search\ResultsGenerator;
+use CultuurNet\UDB3\Search\SearchServiceInterface;
+use CultuurNet\UDB3\StringLiteral;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+final class RejectPlaceDeprecatedType extends AbstractCommand
+{
+    private SearchServiceInterface $searchService;
+
+    public function __construct(
+        CommandBus $commandBus,
+        SearchServiceInterface $searchService
+    ) {
+        $this->searchService = $searchService;
+        parent::__construct($commandBus);
+    }
+
+    public function configure(): void
+    {
+        $this->setName('place:actortype:reject');
+
+        $this->setDescription(
+            'Rejects all places with a given actorType'
+        );
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $output->setVerbosity(OutputInterface::VERBOSITY_VERY_VERBOSE);
+        $logger = new ConsoleLogger($output);
+
+        $deprecatedTypeId = $this->askDeprecatedType($input, $output);
+
+        $resultsGenerator = new ResultsGenerator(
```

Comment:
There is some duplcated code in my PR's, so I thought a trait might be useful. But since these command should probably be removed after they have served their purpose, it might be overkill?